src/main/java/com/box/sdk/BoxFile.java

```diff
@@ -128,6 +128,7 @@
     public static final URLTemplate GET_ALL_FILE_COLLABORATIONS_URL = new URLTemplate("files/%s/collaborations");
     private static final int BUFFER_SIZE = 8192;
     private static final int GET_COLLABORATORS_PAGE_SIZE = 1000;
+    private static final String CLASSIFICATION_TEMPLATE_KEY = "securityClassification-6VMVochwUWo";
```

Comment:
It actually might be helpful to make this `public`, so people can use metadata operations directly if they want to.  Maybe consider moving it to the `Metadata` class so it's not duplicated in both `BoxFile` and `BoxFolder`?